Assignments/Program.cs

```diff
@@ -1 +1,67 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T1 - Number Conversion Game
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Spark;
+
+public class T1 {
+   public static void Main () {
+      List<int> testCases = new () { 0, 1, 4, 9, 10, 11, 16, 25, 36, 49, 50, 81, 100, 109, 125, 216, 343, 729, 1000, 12001, 66000, 123450, 1000909, 987654321, 100000001 };
+      Console.WriteLine ("\x1B[4m" + "Decimal to Binary & Hexadecimal:" + "\x1B[0m");
+      for (int i = 0; i < testCases.Count; i++) {
+         var item = testCases[i];
+         Console.WriteLine ($"\n{i + 1}) {item}");
+         PrintResult (item);
+      }
+      Console.Write ("\nTry Yourself! Enter input number: ");
+      if (int.TryParse (Console.ReadLine (), out int num)) {
+         PrintResult (num);
+      }

```

Comment:
Single statement don't need { }
```suggestion
      if (int.TryParse (Console.ReadLine (), out int num))
         PrintResult (num);

```